VoodooI2CHID/Info.plist

```diff
@@ -89,6 +89,10 @@
 					<integer>13</integer>
 				</dict>
 			</array>
+			<dict>
```

Comment:
Missing `<key>IOPropertyMatch</key>` before it?

Reply:
Good catch :)! It's been added.